sw/device/lib/testing/pinmux_testutils.c

```diff
@@ -0,0 +1,103 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/pinmux_testutils.h"
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/lib/dif/dif_base.h"
+#include "sw/device/lib/dif/dif_pinmux.h"
+#include "sw/device/lib/runtime/hart.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+#define PINMUX0_BASE_ADDR TOP_EARLGREY_PINMUX_AON_BASE_ADDR
+
+#define NUM_GPIO 32
```

Comment:
This is already defined in the autogenerated `gpio_regs.h` header as `GPIO_PARAM_REG_WIDTH`. Could you just use that macro instead of defining a new one?

Reply:
As discussed offline, this was a mistake. Pls disregard.